tests/test_wait_for_it_to_not_raise_an_exception.py

```diff
@@ -0,0 +1,44 @@
+import time
+import unittest
+
+import wait_for_it_to
+
+try:
+    a = TimeoutError()
+except NameError:
+    TimeoutError = wait_for_it_to.TimeoutError
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redefining built-in 'TimeoutError'](https://app.codacy.com/app/studioj/wait_for_it/pullRequest?prid=4638435)